Propose Link for Sovereign Region to clarify doubt

chunsiongtan opened the pull request:
This is the first time I'm reading this documentation, and it might be confusing as to us "sovereign regions" refers to specific 3 data centers for Microsoft Azure. 

As a customer or partner, it might be assumed  "sovereign regions" refers to any specific datacenter and their country of establishment. Example the entire phrase can be mistaken as such 

"When you set a region, remember that Azure Cosmos DB respects sovereign and government clouds. That is, if you create an account in a [SouthEastAsia], you cannot replicate out of that [SouthEastAsia]. Similarly, you cannot enable replication into other sovereign locations from an outside account. "

I'm proposing a hyperlink to our official page, in case customers / partners need more details on what is " "sovereign regions"

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

chunsiongtan committed:
Propose Link for Sovereign Region to clarify doubt

This is the first time I'm reading this documentation, and it might be confusing as to us "sovereign regions" refers to specific 3 data centers for Microsoft Azure. 

As a customer or partner, it might be assumed  "sovereign regions" refers to any specific datacenter and their country of establishment. Example the entire phrase can be mistaken as such 

"When you set a region, remember that Azure Cosmos DB respects sovereign and government clouds. That is, if you create an account in a [SouthEastAsia], you cannot replicate out of that [SouthEastAsia]. Similarly, you cannot enable replication into other sovereign locations from an outside account. "

I'm proposing a hyperlink to our official page, in case customers / partners need more details on what is " "sovereign regions"
